README.md

```diff
@@ -4,15 +4,27 @@ Code style used by AlpineLab in all our projects.
 
 ## Usage
 
-### Rubocop integration
-
-* Use this gem as a dependency, or just install it;
-* Update Rubocop configuration with:
-  ``` yaml
-  inherit_gem:
-    alpinelab-codestyle:
-      - config/default.yml
-  ```
+1. Add this gem to your `Gemfile`:
+
+    ```ruby
+    group :development do
+      gem "alpinelab-codestyle"
```

Comment:
Not sure, but maybe we want to suggest adding the version specification too?
```
gem "alpinelab-codestyle", "~> 0.3.0"
```

For example, and maybe later explain about the versioning?